pkg/tfpfbridge/tests/internal/cmd/pulumi-resource-tls/schema.json

```diff
@@ -74,15 +74,7 @@
                     "type": "string"
                 }
             },
-            "type": "object",
-            "required": [
-                "fromEnv"
-            ],
-            "language": {
-                "nodejs": {
-                    "requiredInputs": []
-                }
-            }
+            "type": "object"
```

Comment:
Looks like fromEnv stopped being recognized as required, need to double check why.

Reply:
OK this is now simply marked `Optional: true` in the upstream version of the tls provider and has nothing to do with the framework changes being made.